experimental/table/server/common/src/main/java/alluxio/table/common/udb/UdbConfiguration.java

```diff
@@ -12,24 +12,69 @@
 package alluxio.table.common.udb;
 
 import alluxio.table.common.BaseConfiguration;
+import alluxio.table.common.ConfigurationUtils;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.Collections;
+import java.util.HashMap;
 import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /**
  * This represents a configuration of the UDB.
  */
 public class UdbConfiguration extends BaseConfiguration<UdbProperty> {
   private static final Logger LOG = LoggerFactory.getLogger(UdbConfiguration.class);
 
+  protected final Map<String, Map<String, String>> mMountOptions;
+
+  protected UdbConfiguration() {
+    super();
+    mMountOptions = new HashMap<>();
+  }
+
   /**
    * Creates an instance.
    *
    * @param values the map of values
    */
   public UdbConfiguration(Map<String, String> values) {
     super(values);
+    mMountOptions = new HashMap<>(values.size());
+    String mountPrefix = ConfigurationUtils.MOUNT_PREFIX;
+    Pattern datePatt = Pattern.compile("(\\(.*\\))\\.(.+?)");
```

Comment:
Make this `private static` so we don't compile every time?

Reply:
done